data-collection/docker-compose.yml

```diff
@@ -19,18 +19,6 @@ services:
     volumes:
       - ./volumes/rabbitmq:/var/lib/rabbitmq
 
-  db:
```

Comment:
Can we leave this block, but commented? And with some nice explanation? So that, in the future, someone can decide whether to use the docker mysql, or an external one.

Reply:
Can we put our specific docker on a separate file, actually? So that this docker-compose is ready for others who want to replicate? And ours, specific to our config, we have a different name, like _serg-docker-compose.yml_?

Reply:
We could do that, but is it necessary?